packages/navlist/src/FzNavlist.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts" setup>
+import { FzRouterNavlinkProps, FzNavlink } from '@fiscozen/navlink'
+import { FzCollapse } from '@fiscozen/collapse'
+import { FzNavlistProps, FzNavlistSub } from './types'
+
+defineProps<FzNavlistProps>()
+
+const isSubMenu = (item: FzRouterNavlinkProps | FzNavlistSub): item is FzNavlistSub => {
```

Comment:
I think this type guard belongs in `types.ts`

Reply:
fair, usually I tend not to have runtime definitions in "types" files but this is strictly related to typings